examples/language_modeling-tf.ipynb

```diff
@@ -0,0 +1,1391 @@
+      "WARNING:tensorflow:The parameter `return_dict` cannot be set in graph mode and will always be set to `True`.\n",
+      "1201/1201 [==============================] - ETA: 0s - loss: 1.8931 - loss_loss: 1.8931WARNING:tensorflow:The parameters `output_attentions`, `output_hidden_states` and `use_cache` cannot be updated when calling a model.They have to be set to True/False in the config object (i.e.: `config=XConfig.from_pretrained('name', output_attentions=True)`).\n",
+      "WARNING:tensorflow:The parameter `return_dict` cannot be set in graph mode and will always be set to `True`.\n",
+      "1201/1201 [==============================] - 121s 97ms/step - loss: 1.8931 - loss_loss: 1.8931 - val_loss: 1.7086 - val_loss_loss: 1.7086\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<keras.callbacks.History at 0x7fe6d1a70f40>"
+      ]
+     },
+     "execution_count": 43,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.fit(train_set, validation_data=validation_set, epochs=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "KDBi0reX3l_g"
+   },
+   "source": [
+    "Like before, we can evaluate our model on the validation set and compute perplexity. The perplexity is much lower than for the CLM objective because for the MLM objective, we only have to make predictions for the masked tokens (which represent 15% of the total here) while having access to the rest of the tokens. It's thus an easier task for the model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "metadata": {
+    "id": "4hSaANqj3l_g",
+    "outputId": "eeeb8727-2e27-4aeb-ac71-c98123214661"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "125/125 [==============================] - 4s 32ms/step - loss: 1.6847 - loss_loss: 1.6847\n",
+      "Perplexity: 5.39\n"
+     ]
+    }
+   ],
+   "source": [
+    "import math\n",
+    "\n",
+    "eval_results = model.evaluate(validation_set)[0]\n",
+    "print(f\"Perplexity: {math.exp(eval_results):.2f}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "wY82caEX3l_i"
+   },
+   "source": [
+    "You can now upload the result of the training to the Hub just as before, just execute this instruction:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 45,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "ValueError",
+     "evalue": "You must login to the Hugging Face hub on this computer by typing `transformers-cli login` and entering your credentials to use `use_auth_token=True`. Alternatively, you can pass your own token as the `use_auth_token` argument.",
```

Comment:
ah it seems you ran into an error because you were not authenticated here? 
```
ValueError: You must login to the Hugging Face hub on this computer by typing `transformers-cli login` and entering your credentials to use `use_auth_token=True`. Alternatively, you can pass your own token as the `use_auth_token` argument.
```
With `huggingface-hub==0.0.17` there's a notebook login that @sgugger contributed:

```
from huggingface_hub import notebook_login

notebook_login()
```